netbox/users/models/features.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+from django.db import models
+
+from extras.utils import is_taggable
+
+
+class CloningUserMixin(models.Model):
+    """
+    Provides the clone() method used to prepare a copy of existing objects.
+    The same code from netbox/users/models/features.py (CloningMixin) is used here.
+    It was necessary to avoid circular imports.
```

Comment:
It's not acceptable to duplicate code from other modules, as this poses serious challenges to long-term maintainability. If employing the original mixin class results in a circular dependency, some other solution must be identified.

Reply:
One solution is to create a "clone"  method in the ObjectPermission, removing the support for "tags" and "custom_fields" (they do not exist on the Users module).
I tested multiple solutions to avoid duplicate code, but they did not work. All of them caused circular imports.